package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cwlts",
-  "version": "1.15.35",
+  "version": "1.15.35.1",
```

Comment:
@kghose Semver doesn't support the fourth number, and this would be invalid: https://npm.community/t/using-4-digit-as-version/3458
I would change this to 1.15.35-patch.1